google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/spi/v1/Publisher.java

```diff
@@ -17,12 +17,12 @@
 package com.google.cloud.pubsub.spi.v1;
 
 import com.google.api.gax.batching.BatchingSettings;
-import com.google.api.gax.core.ApiFuture;
-import com.google.api.gax.core.ApiFutures;
+import com.google.api.core.ApiFuture;
+import com.google.api.core.ApiFutures;
 import com.google.api.gax.core.FlowControlSettings;
 import com.google.api.gax.core.FlowController;
 import com.google.api.gax.core.RetrySettings;
-import com.google.api.gax.core.SettableApiFuture;
+import com.google.api.core.SettableApiFuture;
```

Comment:
I don't want to do that in this PR: it would mean many unrelated changes. I can do that in a separate PR